webapp/src/index.tsx

```diff
@@ -687,6 +688,12 @@ export default class Plugin {
             const currentChannelId = getCurrentChannelId(store.getState());
             if (currChannelId !== currentChannelId) {
                 currChannelId = currentChannelId;
+
+                // If we haven't retrieved config, user must not have been logged in during onActivate
+                if (!callsConfig(store.getState()).retrieved) {
+                    store.dispatch(getCallsConfig());
+                }
```

Comment:
Is there any benefit in "polluting" the config reducer instead of having a state variable in scope to keep track of whether we retrieved it or not?

Reply:
So yah, that's the benefit to putting the flag in the reducer -- it keeps everything clean and async. I personally don't mind it, and it has the benefit of keeping the logic in one place.

Reply:
Do we have to check the error? I was more like thinking of tracking the attempt, assuming in most cases it will work but do you think it could fail somehow and cause issues?

Reply:
Nothing wrong with your approach really, it just felt weird to me re-using the config object itself to keep track of that.